stable/rbac-manager/values.yaml

```diff
@@ -23,3 +23,13 @@ affinity: {}
 
 podAnnotations: {}
 podLabels: {}
+
+service:
+  enabled: false
+  annotations: {}
```

Comment:
It looks like these values aren't currently used. In the service template, you use the serviceMonitor values (which is what I was expecting)

```suggestion
```